docs/source/preferences/Preferences.rst

```diff
@@ -202,30 +202,14 @@ preferences node directly::
   >>> preferences.set('acme.ui.ratio', 0.33)
   ratio 0.75 0.33
 
-If you always use the same preference node as the root of your preferences you
-can also set the class attribute 'PreferencesHelper.preferences' to be that
-node and from then on in, you don't have to pass a preferences collection in
-each time you create a helper::
-
-  >>> PreferencesHelper.preferences = Preferences(filename='example.ini')
```

Comment:
This mutates class attribute (global state) and it also does not work... `helper.bgcolor` returns `""`, `helper.width` returns `0` (default of Int), etc.